web/packages/api/src/toPolkadotSnowbridgeV2.ts

```diff
@@ -200,3 +201,26 @@ export function claimerFromBeneficiary(assetHub: ApiPromise, beneficiaryAddressH
 export function claimerLocationToBytes(claimerLocation: Codec) {
     return hexToU8a(claimerLocation.toHex())
 }
+
+// Agent creation exports
+export type {
+    AgentCreation,
+    AgentCreationValidationResult,
+    AgentCreationInterface,
+} from "./registration/agent/agentInterface"
+
+export function createAgentCreationImplementation() {
+    return new CreateAgent()
+}
+
```

Comment:
I assume this PR serves as a companion for [#1617](https://github.com/Snowfork/snowbridge/pull/1617).

Since the agent call is for the P→E direction, should we move it to `toEthereumSnowbridgeV2` instead?

Reply:
Moved to Ethereum: https://github.com/Snowfork/snowbridge/pull/1620/commits/a8bd39de2a43bab3546a3f2efeb6e3d363370d93